### ISSUE TITLE
No Pokemons available for transfer

### ISSUE THREAD

**Issue by lorenzo93** [finndev/PokeBuddy#15]
I put the flag on "transfer bad pokemon" but the software is not transferring anything.

My Pokemon Inventory is now full (250/250) and the client is always repeating 
"[Transferring Pokemons] No Pokemons available for transfer. Trying again later..."

I opened the inventory through the android app and I have lots of pokemon with PL100 or less and my best pokemons are average level of 1000 so why pokebuddy is not returning them to professor?

**Comment by Bayramklc** [comment]
phone location off pokemon go account login bad pokemon transfer , evolve and power up

LOCATION OFF pokemon go login

**Comment by Vanethir** [comment]
Sometimes you can just Stop and Start the Bot again and it gets rid of the Pokemon for you.

**But PokeBuddy won't transfer ever low CP Pokemon anyways. A list of Pokemon PokeBuddy won't transfer:**

1. Eevee
2. Exeggcute   
3. Bulbasaur
4. Squirtle
5. Ekans
6. Clefairy (?)
7. Meowth
8. Growlithe
9. Geodude
10. Ponyta

Probably some more. They are just the ones I obviously noticed.

In Settings is a list about which Pokemon to keep and evolve. But the list either doesn't work or  just doesn't show the content.

**Comment by jacearchitect69** [comment]
Adding to the list : (pokemon who are not transferred)
Metapod
Eevee
Ponyta
Sandslash
Scyther
Seaking
Tangela
Tentacool
Venomoth
Weepinbell

**Comment by cercata** [comment]
It seems like a threading issue, I stoped the bot, and now it's transferring all the pokemons, but when running, it does it well sometimes, and sometimes doesn't

**Comment by younggunn** [comment]
every pokemon transfers for me except eevee... Which is weird because it hasn't always done this. Only until today has it not transfered eevee's. Im up to over 50+ Eevee's to manually transfer. 

I know its been stated that the eevee is an issue, but i attempted multiple restart/stops through out the day, so im not so sure that it is a thread issue.

**Comment by yuneedmyname** [comment]
The only Pokemon I can confirm that are not being transferred are Magikarp, Eevee, and Dratini (always ending up with a load of Magikarp and Eevee), though this morning I ran into a problem I hadn't seen (in my short time using it). I noticed that my "inventory" (for Pokemon) was full, so I hit "Stop". As soon as it stopped, it suddenly kicked into transfer mode and got rid of a huge load of Pokemon (including an Omanyte, but I'm not concerned as it still left me a much stronger one).
